packages/grafana-ui/src/components/Forms/RadioButtonList/RadioButtonList.tsx

```diff
@@ -48,7 +48,7 @@ export function RadioButtonList<T>({
 
         return (
           <RadioButtonDot
-            key={itemId}
+            key={`${option.label}-${index}`}
```

Comment:
Does the old code create some issues? We use `uniqueId` in a few other places as well.

Reply:
Yes, see my comment in the corresponding issue https://github.com/grafana/grafana/issues/72729#issuecomment-1663582705
In general, it causes list element re-render and detaches it from DOM. So ClickOutsideWrapper cannot find it anymore and calls onClick() handler (which usually used to close menu, modal, etc).